tests/conftest.py

```diff
@@ -38,6 +39,12 @@ def embed(*args, **kwargs) -> Embeddings:
 
         return Embeddings(embeddings=[[1.0, 1.0]])
 
+    def tokenize(*args, **kwargs) -> Tokens:
+        tokens = [x for x in range(len(args[1].split()))]
+        tokens_strings = args[1].split()
+
+        return Tokens(tokens, tokens_strings)
```

Comment:
```suggestion
        token_strings = args[1].split()
        tokens = [x for x in range(len(token_strings))]

        return Tokens(tokens, token_strings)
```
nit on variable naming (to follow cohere python sdk) and not splitting twice